go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/CiscoDevNet/terraform-provider-iosxr
 
-go 1.20
+go 1.21
```

Comment:
Is there a specific reason you are bumping the go version?

Reply:
reverted to v0.3.2 commit